keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -905,7 +906,11 @@ Major milestones might include:
 
 ### Alpha
 
-Alpha feature was implemented in 1.27.
+Alpha feature was implemented in 1.27: <https://github.com/kubernetes/kubernetes/pull/115220>
+
+### Beta
+
+Add e2e tests(TODO)
```

Comment:
Maybe you should fill this out with more details.  What tests are you going to add?
/lgtm cancel

Reply:
My initial thought is 
```
[serial] test

1. Choose a worker node
2. Config `Parallel Image Pull Limit` to be 1(default)
3. Pre-delete images(image-1/2/3/4/5/6) on node
4. Deploy 1 pod and then 5 pods with all image-1 and get the image to pull time including waiting.
  a. The first pod pull duration (duration-A1)
  b. Every pod pull duration of the 5 pods later(duration-B1)
5. Pre-delete images(image-1/2/3/4/5/6) on node
6. Config `Parallel Image Pull Limit` to be 5
7. Deploy 1 pod and then 5 pods with all image-1 and get the image to pull time including waiting.
  a. The first pod pull duration A2 (Duration-A2 should be similar with the duration-A1)
  b. Every pod pull duration of the 5 pods later(The duration-B2 should be similar with Duration A1) less than duration-B1/3(we may leave some space here.)

```

This just check if this can support parallel image pull and limit. 

As this is time based, I wondered if we need to add a more complex one.

Reply:
https://github.com/kubernetes/kubernetes/pull/18986 GCE e2e default will use `--serialize-image-pulls=false` if I understand correctly.

Reply:
Another test is mentioned by  Ruiwen above 

https://github.com/kubernetes/enhancements/blob/1dd0bad29863ef89213c53f8426414fe9c045a5c/keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md?plain=1#L384-L386

Reply:
Updated. 
After implementation, I will update both later.